### PR TITLE
Devdocs-2365 get product by

### PR DESCRIPTION
**Pull request by traci707**
Removed price_list_id query parameter from Get a Product endpoint because it no longer works.
See [DEVDOCS-2365](https://jira.bigcommerce.com/browse/DEVDOCS-2365)

### CODE REVIEW

File: reference/channels.v3.yml [hunk 1]
```diff
@@ -452,11 +452,9 @@ paths:
                 1.product_id: error.path.missing
```
Comment (aglensmith) [inline reference/channels.v3.yml:452]: Remove the changes to this file from the pr.

File: reference/marketing.v2.yml [hunk 2]
```diff
@@ -2,61 +2,37 @@ swagger: '2.0'
 info:
```
Comment (aglensmith) [inline reference/marketing.v2.yml:2]: remove the changes to this file from the pr.

**Review by aglensmith — CHANGES_REQUESTED** [review]

File: reference/channels.v3.yml (hunk 1 above)
Comment (traci707) [reply reference/channels.v3.yml:452]: When I remove it, I receive a message saying the yml file is invalid.

File: reference/marketing.v2.yml (hunk 2 above)
Comment (traci707) [reply reference/marketing.v2.yml:2]: This change also invalidates the file.

**Pull request closed without merging**